docs/reference/extensions/dotnet-extensions.rst

```diff
@@ -0,0 +1,120 @@
+.. _reference-dotnet-extensions:
+
+.NET extensions
+===============
+
+The .NET extensions provide support for building and running .NET application snaps. They are versioned extensions,
+which means that you can specify which extension you want to use based on the .NET major version your application
+targets. The available extensions are:
+
+* ``dotnet8``: For applications targeting .NET 8 (``net8.0``)
+* ``dotnet9``: For applications targeting .NET 9 (``net9.0``)
+* ``dotnet10``: For applications targeting .NET 10 (``net10.0``)
+
+The .NET extensions are compatible with the core24 base.
+
+Included parts
+--------------
+
+The .NET extensions add the following parts to the project file:
+
+* ``prereqs``: A part that installs the prerequisite packages for running .NET applications. The installed packages are:
+
+   * ``libicu74``
+   * ``libssl3t64``
+   * ``libunwind8``
+   * ``liblttng-ust1t64``
+   * ``libbrotli1``
```

Comment:
```suggestion
  * ``libicu74``
  * ``libssl3t64``
  * ``libunwind8``
  * ``liblttng-ust1t64``
  * ``libbrotli1``
```

With the extra space, they're rendering as block quotes. rST is fiddly. :(

Reply:
So _that_ is how you fix this! Thanks! 😄